content/en/docs/concepts/scheduling/kube-scheduler.md

```diff
@@ -173,10 +183,161 @@ kube-scheduler has a default set of scheduling policies.
   that don't have Pods for the service already assigned there. The overall outcome is
   that the Service becomes more resilient to a single Node failure.
 
-- `CalculateAntiAffinityPriorityMap`: This policy helps implement
-  [pod anti-affinity](/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity).
-
-- `EqualPriorityMap`: Gives an equal weight of one to all nodes.
+- `EqualPriority`: Gives an equal weight of one to all nodes.
+
+- `EvenPodsSpreadPriority`: Implements preferred
+  [pod topology spread constraints](/docs/concepts/workloads/pods/pod-topology-spread-constraints/).
+
+### Scheduling profiles
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+Scheduling profiles allow more flexibility when configuring a scheduler.
```

Comment:
>  Scheduling profiles allow more flexibility when configuring a scheduler.

If this means “more than the old code”, consider rewording. Ideally, you could keep the same text for the alpha, beta and the GA feature (possibly dropping a bit about “enable feature gate _FooBar_ first”).

Reply:
I'm happy to just drop the sentence. On another note, there is no feature gate. Users just have to use the v1alpha2 API.

Reply:
Removed